hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -212,6 +212,32 @@ def _bank_add_signals(
     }
 
 
+@bp.route("/bank/<bank_name>/content/<content_id>", methods=["PUT"])
+def bank_update_content(bank_name: str, content_id: int):
+    """
+    Update the metadata for a banked content item.
+    """
+    storage = persistence.get_storage()
+    bank = storage.get_bank(bank_name)
+    if not bank:
+        abort(404, f"bank '{bank_name}' not found")
+    content = storage.bank_content_get([content_id])
+    if not content:
+        abort(404, f"content '{content_id}' not found")
+    content = content[0]
+    data = request.get_json()
+
+    try:
+        if "disable_until_ts" in data:
+            content.disable_until_ts = flask_utils.str_to_type(
+                data["disable_until_ts"], int
+            )
+        storage.bank_content_update(content)
+    except Exception as e:
+        abort(400, *e.args)
```

Comment:
blocking: Some of these exceptions would be the servers fault (500). I don't think we should capture every exception here to try and distinguish. Can you only capture the exceptions you know to be the client's fault?